cvat-cli/src/cvat_cli/parser.py

```diff
@@ -369,6 +369,21 @@ def make_cmdline_parser() -> argparse.ArgumentParser:
         help="time interval between checks if archive processing was finished, in seconds",
     )
 
+    #######################################################################
+    # Auto-annotate
+    #######################################################################
+    auto_annotate_task_parser = task_subparser.add_parser(
+        "auto-annotate", description="Automatically annotate a CVAT task."
```

Comment:
Probably, it should be clarified, that this command doesn't just execute a task auto-annotation on the remote server, but runs a local script instead (though, a command for the first would also be nice to have as an option).

Reply:
Yeah, that's a fair point. I'm thinking of maybe renaming this command to `auto-annotate-local` to better underscore the distinction (and perhaps `auto-annotate` can run server-side auto-annotation instead).

Reply:
The current name is good enough (though I'd like it to be 1 word somehow), we can just extend the description.

Reply:
Description extended.